dascore/io/dasdae/utils.py

```
@@ -347,36 +329,120 @@ def _get_coord_summary_from_node(coord_node, dims):
+
+
+def _decode_legacy_attr_value(value):
+    """Decode legacy DASDAE attrs still used by the shipped fixture files."""
+    if value.__class__.__name__ == "Empty":
+        return ""
+    if isinstance(value, np.ndarray) and not value.shape:
+        value = np.asarray([value])[0]
+    if isinstance(value, np.bytes_ | bytes):
+        raw = bytes(value)
+        try:
+            value = pickle.loads(raw, encoding="latin1")
```

Ruff (0.15.7)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)